.github/workflows/build.yml

```diff
@@ -437,7 +437,9 @@ jobs:
       - name: Post sticky pull request comment
         uses: marocchino/sticky-pull-request-comment@v2
         with:
-          recreate: true
+#         changed to append while we have aks and paas review apps
+          append: true
```

Comment:
You can use the `header` argument so it creates 2 separate sticky comments as per:
https://github.com/DFE-Digital/schools-experience/blob/master/.github/workflows/build.yml#L486

Reply:
thanks, updated.